docs/data/scheduler/event-calendar/event-calendar.md

```diff
@@ -19,6 +19,72 @@ This package is not published yet.
 
 {{"demo": "FullEventCalendar.js", "bg": "inline", "defaultCodeOpen": false}}
 
+## Recurrence
+
+Recurring events use an `rrule` object to define their repeating pattern.
+The scheduler expands recurring events only for the visible range, keeps the original duration and handles all-day and multi-day spans.
+
+{{"demo": "Recurrence.js", "bg": "inline", "defaultCodeOpen": false}}
+
+### Supported recurrence patterns
+
+We use a typed object (`RRuleSpec`) instead of the raw RRULE string, but it mirrors a subset of the iCalendar RRULE (RFC 5545).
+
+In the `rrule` object, `freq` sets the base frequency of the pattern: `'DAILY'` | `'WEEKLY'` | `'MONTHLY'` | `'YEARLY'`.
+Besides, `interval` is the step between occurrences (defaults to 1) and selectors are optional filters like `byDay` or `byMonthDay`, that refine the base frequency.
+
+Below are the shapes we support, with small examples.
+
+- Daily (`freq: 'DAILY'`), no extra fields required.
+
+```tsx
```

Comment:
The identation of most code blocks are wrong.
The first tick should be at the level of the content (not the dash) so that it's rendered aligned